client/layout/masterbar/drafts.jsx

```diff
@@ -0,0 +1,136 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { getSelectedSite } from 'state/ui/selectors';
+import Popover from 'components/popover';
+import Count from 'components/count';
+import { getMyPostCounts } from 'state/posts/counts/selectors';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import {
+	getSitePostsForQueryIgnoringPage,
+	isRequestingSitePostsForQuery
+} from 'state/posts/selectors';
+import Draft from 'my-sites/draft';
+import QueryPosts from 'components/data/query-posts';
+import QueryPostCounts from 'components/data/query-post-counts';
+import Button from 'components/button';
+import Site from 'blocks/site';
+import { getCurrentUserId } from 'state/current-user/selectors';
+import sitesList from 'lib/sites-list';
+
+const sites = sitesList();
+
+const MasterbarDrafts = React.createClass( {
+	propTypes: {
+		user: React.PropTypes.object,
+		sites: React.PropTypes.object,
+		isActive: React.PropTypes.bool,
+		className: React.PropTypes.string,
+		tooltip: React.PropTypes.string,
+		// connected props
+		selectedSite: React.PropTypes.object,
+	},
+
+	getInitialState() {
+		return {
+			showDrafts: false
+		};
+	},
+
+	toggleDrafts( event ) {
+		event.stopPropagation();
+		event.preventDefault();
+		this.setState( {
+			showDrafts: ! this.state.showDrafts
+		} );
+	},
+
+	closeDrafts() {
+		this.setState( { showDrafts: false } );
+	},
+
+	render() {
+		const site = this.props.selectedSite;
```

Comment:
The selected site should persist as you navigate around. We removed the lastSelectedSite selector, right?